src/development/stack.yml

```diff
@@ -365,6 +368,7 @@ services:
     - /var/run/docker.sock:/var/run/docker.sock
     - ./certificates/:/etc/traefik/acme/
     - ./configurations/traefik/dynamic.yml:/dynamic.yml:ro #DARGSTACK-REMOVE
+    # - ./configurations/traefik/plugins-local:/plugins-local
```

Comment:
Can this line be removed?

Reply:
Good find! I'll remove it as we don't use local traefik plugins anymore. 👍